athena-federation-sdk/src/test/java/com/amazonaws/athena/connector/lambda/serde/v2/ListTablesRequestSerDeTest.java

```diff
@@ -43,7 +44,8 @@
     public void beforeTest()
             throws IOException
     {
-        expected = new ListTablesRequest(federatedIdentity, "test-query-id", "test-catalog", "test-schema");
+        expected = new ListTablesRequest(federatedIdentity, "test-query-id", "test-catalog",
+                "test-schema", null, NULL_PAGE_SIZE);
```

Comment:
can we use real values here?

Reply:
`null`, and `-1` are real values that will be passed from the CP. `null` indicates that this is the first round of pagination, while `-1` indicates that the list of tables can be infinite in size or not-paginated.

Reply:
keep in mind we don't normally discuss CP here, as Athena isn't the only thing to call these APIs.